.github/workflows/daily-dev-bump.yaml

```diff
@@ -75,6 +75,15 @@ jobs:
             dart tool/update_version.dart auto --type major
             dart tool/update_version.dart auto --type dev
           elif [ "$UPDATE_TYPE" == "dev" ]; then
+            if ! echo "$ORIGINAL_VERSION" |grep -Eq "\-dev\.[0-9]+" ; then
```

Comment:
Nit: can this whitespace be cleaned up a tiny bit to something like this?

`if ! echo "$ORIGINAL_VERSION" | grep -Eq "\-dev\.[0-9]+"; then`